naccs_bot.py

```diff
@@ -188,21 +189,26 @@ def get_ongoing_matches(channel_id):
             response_image = (streams_json["payload"][x]["stream"]["channelLogo"])
             response_comp_name = (streams_json["payload"][x]["competitionName"])
             response_channel_url = (streams_json["payload"][x]["stream"]["channelUrl"])
-            response_competition_name = (streams_json["payload"][x]["factionNickname"])
+            response_faction_name = (streams_json["payload"][x]["factionNickname"])
             response_viewers = (streams_json["payload"][x]["stream"]["viewers"])
             active_streams[response_nick] = response_nick
+
+            channel = client.get_channel(LEAGUE_STREAMS)
+            embed = Embed(title=response_nick, url=response_channel_url, description=response_comp_name, color=0x5e7aac)
+            embed.set_thumbnail(url=response_image)
+            embed.set_author(name="A League Stream is Live!", icon_url="https://naccs-s3.s3.us-east-2.amazonaws.com/static/assets/headerlogo_small.png")
```

Comment:
Not all streams will be a league stream right?

Reply:
Yes that is correct. I totally missed that. I will fix that as well as add some checks for those API issues later